Source/Resources/Content/TeamsPSAdmin/TeamsAppPermissionPolicy/Global.json

```diff
@@ -1,6 +1,7 @@
 {
   "defaultCatalogApps": [],
   "defaultCatalogAppsType": "BlockedAppList",
+  "description":"What is it: Defines application policies for Teams. \n Learn More: https://docs.microsoft.com/en-us/microsoftteams/enable-features-office-365",
```

Comment:
**What does this do?**
Defines application policies for Teams. This is the default that Microsoft provides. The baseline includes this as a starting point.

**Why should you use this?**
It is recommended so it may be applied to all tenants. 

**What is the end user impact?**
N/A

**Learn more**
[Manage app permission policies in Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/teams-app-permission-policies)